polaris-tokens/scripts/getKeyframeNames.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path');
 
-const tokenGroupsDir = path.join(__dirname, '../../src/tokens/token-groups');
+const tokenGroupsDir = path.join(__dirname, '../src/token-groups');
```

Comment:
Same as above, can we move this to `src/utilities` and update to directly import the `motion` token group?